test/test_handlers.py

```diff
@@ -53,6 +53,21 @@ def _sample():
     assert_allclose(xs, ys, atol=1e-6)
 
 
+def test_nested_seeding():
+    def fn(rng_1, rng_2, rng_3, sample_x=False):
```

Comment:
what is the `sample_x` for?

Reply:
My bad, there was a case I wanted to test against, but I am not changing sample_x. Will figure out a better way to test it, and fix.